<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1109</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>550</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>test</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCurrentCfgFileName">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>600</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1109</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="actionCreateFile"/>
    <addaction name="actionOpenFile"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <addaction name="actionEditCfgFile"/>
    <addaction name="actionShowLogs"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="text">
    <string>Открыть...</string>
   </property>
  </action>
  <action name="actionCreateFile">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="actionEditCfgFile">
   <property name="text">
    <string>Редактировать cfg файл</string>
   </property>
  </action>
  <action name="actionShowLogs">
   <property name="text">
    <string>Посмотреть лог</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
